doc/nrf/external_comp/memfault.rst

```diff
@@ -27,21 +27,22 @@ Before you start the |NCS| integration with Memfault, make sure that the followi
 
 * :ref:`Installation of the nRF Connect SDK <installation>`.
 * :ref:`Setup of nRF9160 DK <ug_nrf9160_gs>`.
-* Sign up in the `Memfault registration page`_ and `create a new project in Memfault`_.
+* Sign up in the `Memfault registration page`_.
+* `Create a Memfault project <create a new project in Memfault_>`_.
   You will be directed to the Integration guide for Memfault.
   You receive the ``project key`` here to set in the :kconfig:option:`CONFIG_MEMFAULT_NCS_PROJECT_KEY` Kconfig option.
```

Comment:
```suggestion
  You will be directed to the Integration guide for Memfault.

  You receive the ``project key`` here to set in the :kconfig:option:`CONFIG_MEMFAULT_NCS_PROJECT_KEY` Kconfig option.
```
I think this might improve readability.